packages/react-provider/.babelrc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "plugins": ["module:@fluentui/babel-make-styles", "annotate-pure-calls", "@babel/transform-react-pure-annotations"]
+  "presets": ["@griffel"],
```

Comment:
Griffel exposes Babel preset instead of a Babel plugin. Currently it's only difference in config

Reply:
hmm not sure it's good idea to encapsulate 2 additional plugins under `@griffel`.  I'd keep `griffel` preset focused on its main purpose - "compiling" css in js. WDYT?

Reply:
🙈 nevermind, note to myself: "Martin read comments only in diff/files tab, you're welcome"